stack-citationapi/recipes/deploy-publicapi.rb

```diff
@@ -0,0 +1,32 @@
+node['deploy'].each do |application, deploy|
+  case application
+  when 'easybib'
+    next unless allow_deploy(application, 'easybib', 'nginxphpapp')
+  when 'citation_apis'
```

Comment:
This should be removed, or the public api would be redeployed during a citation api deploy